<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1377</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #fff;</string>
  </property>
  <widget class="QFrame" name="display_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0" colspan="4">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QListWidget" name="list_rappels">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>501</width>
         <height>500</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font-weight: bold;
background: #fffffe;</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
      </widget>
      <widget class="QWidget" name="display_widget" native="true">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>100</y>
         <width>741</width>
         <height>501</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_creation">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>330</y>
          <width>231</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #1a1a1a;
border-radius: 6px;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_status">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>330</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #1a1a1a;
border-radius: 6px;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QTextBrowser" name="label_commentaire">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>160</y>
          <width>651</width>
          <height>141</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
border: 1px solid #28282b;
color: #28282b;
</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:8px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_patient">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>80</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
border: 1px solid #28282b;
color: #28282b;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_motif">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>80</y>
          <width>401</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
border: 1px solid #28282b;
color: #28282b;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>420</y>
          <width>741</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="tel_btn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>41</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Appeler</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="email_btn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>41</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="done_btn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>41</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Termin√©</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QLabel" name="message">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>501</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background: transparent;
color: #FF0000;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="cancel_btn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background: transparent;
border: 1px solid #28282b;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>message</zorder>
      <zorder>list_rappels</zorder>
      <zorder>display_widget</zorder>
      <zorder>cancel_btn</zorder>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="after_btn">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background: #28282b;  
color: #fefefe;
text-align: center;
font-weight: bold;
border: none;
padding: 8px;  }
QPushButton:hover {
background: transparent;
color: #28282b;
border: 1px solid #28282b;  }
                </string>
      </property>
      <property name="text">
       <string>Apres demain</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="today_btn">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background: #28282b;  
color: #fefefe;
text-align: center;
font-weight: bold;
border: none;
padding: 8px;  }
QPushButton:hover {
background: transparent;
color: #28282b;
border: 1px solid #28282b;  }</string>
      </property>
      <property name="text">
       <string>Aujourdh'ui</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="demain_btn">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
background: transparent;
color: #28282b;
border: 1px solid #28282b;  
text-align: center;
font-weight: bold;
padding: 8px;  }

         </string>
      </property>
      <property name="text">
       <string>Demain</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="date_btn">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background: #28282b;  
color: #fefefe;
text-align: center;
font-weight: bold;
border: none;
padding: 8px;  }
QPushButton:hover {
background: transparent;
color: #28282b;
border: 1px solid #28282b;  }
                
                
                </string>
      </property>
      <property name="text">
       <string>Choisir date</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
